blockchain/types/tx_internal_data_ethereum_access_list.go

```diff
@@ -367,7 +383,7 @@ func (t *TxInternalDataAccessList) String() string {
 	} else {
 		to = fmt.Sprintf("%x", t.GetRecipient().Bytes())
 	}
-	enc, _ := rlp.EncodeToBytes(t)
```

Comment:
Why did you change this argument of EncodeToBytes (from t to tx)?

Reply:
If it encodes a `t`, it only encode fields in txInternalDataAccessList. 

However, Wehn encoding the transaction payload(a.k.a Hex in String()), it needs to additional field like `ethEnvelope || txType || rlp(txInternalDataAccessList)`.

So I implemented encoding / decoding logic in transaction.go(exactly `TxInternalDataSerializer`) and it encodes transaction variable `tx` to get transaction payload.

Reply:
Oh I got it, thanks for your kind comment